doc/source/install.rst

```diff
@@ -72,6 +72,285 @@ and start hacking it with:
+     (synopsis "Typing stubs for html5lib")
+     (description "Typing stubs for html5lib")
+     (license #f)))
+  
+  (define python-types-beautifulsoup4
+    (package
+     (name "python-types-beautifulsoup4")
+     (version "4.12.0.6")
+     (source (origin
+              (method url-fetch)
+              (uri (pypi-uri "types-beautifulsoup4" version))
+              (sha256
+               (base32
+                "0iqkh67sv823df87hxbd1s8izqv77zs14dhk2rp1hh75sf2v4nh4"))))
+     (build-system pyproject-build-system)
+     (propagated-inputs (list python-types-html5lib))
+     (home-page "https://github.com/python/typeshed")
+     (synopsis "Typing stubs for beautifulsoup4")
+     (description "Typing stubs for beautifulsoup4")
+     (license #f)))
+  
+  (define python-doi
+    (package
+     (name "python-doi")
+     (version "0.2.0")
+     (source (origin
+              (method url-fetch)
+              (uri (pypi-uri "python-doi" version))
+              (sha256
+               (base32
+                "16pxc7llqb14f2n5ccd88pz4sygwl51slssqm2g23g8rndpya09f"))))
+     (build-system python-build-system)
+     (native-inputs (list python-coverage
+                          python-flake8
+                          python-pep8
+                          python-pytest
+                          python-pytest-cov
+                          python-pytest-xdist
+                          python-sphinx
+                          python-sphinx-autobuild
+                          python-sphinx-rtd-theme
+                          python-twine
+                          python-wheel))
+     (home-page "https://github.com/papis/python-doi")
+     (synopsis "Python package to work with Document Object Identifier (doi)")
+     (description
+      "Python package to work with Document Object Identifier (doi)")
+     (license #f)))
+  
+  (define python-dominate
+    (package
+      (name "python-dominate")
+      ;; (version "2.6.0")
+      ;; (version "2.7.0")
+      (version "2.8.0")
+      (source (origin
+                (method url-fetch)
+                (uri (pypi-uri "dominate" version))
+                (sha256
+                 (base32
+                  ;; "1r71ny73ws0zf5mcml0x5yfbjhzfkn5id670zv26y2kh4gg2rv3n"
+                  ;; "0jfs0n50h0q50ca3iqx1v01j9ycz6mzd6rrzap8gkswj10v020aj"
+                  "01s0a2zqyni2az4wwmd8rxx9gy5ypkvrmczlf4mn33pqzazc742c"
+                  ))))
+      (build-system pyproject-build-system)
+      (arguments '(
+                   ;; #:test-backend "pytest"
+                   #:tests? #f
+                            ))
+      (home-page "https://github.com/Knio/dominate/")
+      (synopsis
+       "Dominate is a Python library for creating and manipulating HTML documents using an elegant DOM API.")
+      (description
+       "Dominate is a Python library for creating and manipulating HTML documents using
+  an elegant DOM API.")
+      (license #f)))
+  
+  (define python-arxiv2bib
```

Comment:
Papis doesn't use `arxiv2bib` anymore. This should probably be updated?

Reply:
This is papis package definition for 0.13 release. As I understand it is current
stable version, right? At that point it was still rely on arxiv2bib.

Reply:
Ah, you're right, my bad!